dhcpv4/option_relay_agent_information.go

```diff
@@ -4,6 +4,14 @@ import (
 	"fmt"
 )
 
+type raiValue struct {
```

Comment:
NIT: can this be moved close to raiSubOptionCode?

Reply:
will do